GVFS/GVFS/RepairJobs/GitConfigRepairJob.cs

```diff
@@ -23,7 +23,7 @@ public override IssueType HasIssue(List<string> messages)
         {
             GitProcess git = new GitProcess(this.Enlistment);
             GitProcess.Result result = git.GetOriginUrl();
-            if (result.HasErrors)
+            if (result.ExitCodeIsFailure)
             {
                 if (result.Errors.Length == 0)

```

Comment:
I am marking this as approved so that I don't prevent the changes from going in when they're ready, but I will still take a look at the new commit when it's ready.